templates/showroom.php

```diff
@@ -1,37 +1,45 @@
-<ul>
-  <?php while (have_rows('field_showroom_window', 'product_cat_' . $term_id)): the_row(); ?>
-  <li>
-    <div class="showroom">
-      <div class="scene">
-        <div class="image">
+<?php
+
+use Netzstrategen\WooCommerceShowroom\Plugin;
+
+$image_attr = Plugin::isLazyLoadActive() ? ' data-no-lazy="1"' : '';
+?>
+
+<div class="showroom gallerya gallerya--slider">
+  <ul class="js-gallerya-slider js-gallerya-lightbox clearfix">
+    <?php while (have_rows('field_showroom_window', 'product_cat_' . $term_id)): the_row(); ?>
+      <li class="showroom-item clearfix">
+        <figure class="gallerya__image">
           <?php $image = get_sub_field('field_showroom_image'); ?>
-          <img src="<?= $image['url'] ?>" alt="<?= $image['alt'] ?>" title="<?= $image['title'] ?>" />
-        </div>
-        <div class="description">
-          <?= get_sub_field('field_showroom_description'); ?>
-        </div>
-      </div>
-      <div class="related-products">
+          <img class="showroom-item-image" src="<?= $image['url'] ?>" alt="<?= $image['alt'] ?>" title="<?= $image['title'] ?>" <?= $image_attr ?> />
+          <?php if (!empty($caption = get_sub_field('field_showroom_description'))): ?>
+            <figcaption class="showroom-item-description"><?= $caption ?></figcaption>
+          <?php endif; ?>
+        </figure>
         <?php
         $related_products = get_sub_field('field_showroom_related_products');
-        if ($related_products) :
-          foreach ($related_products as $related_product_id):
-        ?>
-          <?php $product = wc_get_product($related_product_id); ?>
-          <a href="<?= $product->get_permalink() ?>" title="<?= $product->get_title(); ?>">
-            <div class="image">
-              <?= $product->get_image() ?>
-            </div>
-            <div class="product-name">
-              <?= $product->get_title(); ?>
-            </div>
-          </a>
-        <?php
-          endforeach;
-        endif;
-        ?>
-      </div>
-    </div>
-  <li>
-  <?php endwhile; ?>
-</ul>
+        if ($related_products) : ?>
+          <div class="showroom-related-products">
+            <ul>
+              <?php foreach ($related_products as $related_product_id): ?>
+              <li class="showroom-related-product">
+                <div class="showroom-related-product-image">
+                  <?php $product = wc_get_product($related_product_id); ?>
```

Comment:
Move this line below the foreach to keep the code logic separated from the output.